rasa/core/policies/memoization.py

```diff
@@ -45,44 +65,42 @@ class MemoizationPolicy(Policy):
     training stories for this, use AugmentedMemoizationPolicy.
     """
 
-    ENABLE_FEATURE_STRING_COMPRESSION = True
+    @staticmethod
+    def get_default_config() -> Dict[Text, Any]:
+        """Returns the default config (see parent class for full docstring)."""
+        # please make sure to update the docs when changing a default parameter
+        return {
+            "enable_feature_string_compression": True,
+            "use_nlu_confidence_as_score": False,
+            POLICY_PRIORITY: MEMOIZATION_POLICY_PRIORITY,
+            POLICY_MAX_HISTORY: DEFAULT_MAX_HISTORY,
+        }
 
-    USE_NLU_CONFIDENCE_AS_SCORE = False
+    required_packages = []
 
-    @staticmethod
-    def _standard_featurizer(
-        max_history: Optional[int] = None,
-    ) -> MaxHistoryTrackerFeaturizer:
+    def _standard_featurizer(self) -> MaxHistoryTrackerFeaturizer:
         # Memoization policy always uses MaxHistoryTrackerFeaturizer
         # without state_featurizer
         return MaxHistoryTrackerFeaturizer(
-            state_featurizer=None, max_history=max_history
+            state_featurizer=None, max_history=self.config[POLICY_MAX_HISTORY]
         )
 
     def __init__(
         self,
+        config: Dict[Text, Any],
+        model_storage: ModelStorage,
+        resource: Resource,
+        execution_context: ExecutionContext,
         featurizer: Optional[TrackerFeaturizer] = None,
-        priority: int = MEMOIZATION_POLICY_PRIORITY,
-        max_history: Optional[int] = DEFAULT_MAX_HISTORY,
         lookup: Optional[Dict] = None,
-        **kwargs: Any,
     ) -> None:
-        """Initialize the policy.
-
-        Args:
-            featurizer: tracker featurizer
-            priority: the priority of the policy
-            max_history: maximum history to take into account when featurizing trackers
-            lookup: a dictionary that stores featurized tracker states and
-                predicted actions for them
-        """
+        """Initialize the policy."""
         if not featurizer:
-            featurizer = self._standard_featurizer(max_history)
+            featurizer = self._standard_featurizer()
```

Comment:
I actually made that part of the super class so that you don't need to take care of it 👍🏻